setup.py

```diff
@@ -9,8 +9,11 @@
 URL = 'https://github.com/pangeo-data/xESMF'
 LICENSE = 'MIT'
 PYTHON_REQUIRES = '>=3.6'
-USE_SCM_VERSION = {'version_scheme': 'post-release', 'local_scheme': 'dirty-tag'}
-SETUP_REQUIRES = ['setuptools_scm', 'setuptools>=30.3.0']
+USE_SCM_VERSION = {
+    "write_to": "xesmf/_version.py",
```

Comment:
The version file is much faster than `pkg_resources` b/c, which can get really slow depending on the packages installed on your env.